client/src/pages/admin/review-portfolios/index.tsx

```diff
@@ -1,3 +1,151 @@
+import {
+  Container,
+  Flex,
+  Heading,
+  Box,
+  Text,
+  Textarea,
+  Button,
+} from '@chakra-ui/react'
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbLink,
+  BreadcrumbSeparator,
+} from '@chakra-ui/react'
+import { ChevronRightIcon } from '@chakra-ui/icons'
+import { Navbar } from '../../../components'
+import { useState } from 'react'
+import Head from 'next/head'
+
 export default function ReviewPortfolios(): JSX.Element {
-  return <div>Review Portfolios</div>
+  const [disableButton, setDisableButton] = useState<boolean>(true)
+
+  function handleInputChange(e: { target: { value: string | number } }) {
```

Comment:
- Hey, I guess this is more generic function name, you can name this function something like `addReviewComment` for better readability.
- Also, we can `type` the return type of the function, as its a good practice
```suggestion
  function addReviewComment(e: { target: { value: string | number } }) : void {
```

Reply:
Much better now. Thanks